example/provider-local/seed-kind-ha/local-mz/kustomization.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+- ../base
+- seed.yaml
```

Comment:
Instead of having an own `seed.yaml` here, can you follow the same approach as in https://github.com/gardener/gardener/blob/66b1975adf5d6656930d1273c150792028646ad5/example/provider-local/seed-kind2/local/kustomization.yaml#L6-L14?